model-optimizer/mo/front/tf/extractor.py

```diff
@@ -38,7 +38,7 @@ def create_tf_edge(src_node_id: str, dst_node_id: str, in_port: int):
         'in': in_port,
         'out': src_port,
         # debug anchor for a framework name, out port and tensor name
-        'fw_tensor_debug_info': [(src_node_id, src_port, tensor_name)],
+        'fw_tensor_debug_info': [(src_node_id, tensor_name)],
```

Comment:
In OV 2021.2 the mapping file for TF is the following:

```xml
       <map>
           <framework name="InceptionV1/InceptionV1/Mixed_5c/Branch_0/Conv2d_0a_1x1/Conv2D" out_port_id="0"/>
           <IR name="InceptionV1/InceptionV1/Mixed_5c/Branch_0/Conv2d_0a_1x1/Conv2D" out_port_id="2" id="277"/>
       </map>
```

so the `out_port_id` is the port number and not the tensor name. So the PR description contradicts with the change.

Reply:
May be we should change the behaviour and align it with all frameworks. @GlebKazantaev , what is the behavior in the serializer which you implemented?

Reply:
Sure, we should produce the same mapping file using nGraph and MO legacy IR generation paths.
So, in nGraph transformation which generates mapping file I implemented next behavior:
```
For each node I iterate over output ports and generate next <map> blocks:
   <fw_name>, <fw_output_port_id>, <ir_name>, <ir_output_port_id>
1. node_name, node_name:output_port_id, node_name, ie_output_port_id
2. For each output port name: extracted_name, port_name, node_name, ie_output_port_id
```
I suggest to look at https://github.com/openvinotoolkit/openvino/pull/5347/files#diff-d5436d7c30193d976e9c350a0d04c208cd1ef95a0c243bf370c3931ae3e3585bR36
it is simple.
Also I validated this approach on e2e and DL Benchmark scopes.

Reply:
offline discussion summary; https://github.com/openvinotoolkit/openvino/pull/5347#discussion_r621257289

Reply:
Changed the logic of mapping generation for kaldi and mxnet according to offline discussion. 
Const naming was fixed.